include/NAS2D/Renderer/Color.h

```diff
@@ -38,7 +55,10 @@ class Color_4ub
 	void alpha(uint8_t alpha);
 
 private:
-	unsigned char mR = 255, mG = 255, mB = 255, mA = 255;
+	unsigned char mR = 255;
+	unsigned char mG = 255;
+	unsigned char mB = 255;
+	unsigned char mA = 255;
```

Comment:
This should probably have been done in a separate commit.

Reply:
Oh, and on the subject of that line, it should probably use type `uint8_t`. That would be more consistent with the constructor parameter types, and accessor method types.

Reply:
> 
> 
> Oh, and on the subject of that line, it should probably use type `uint8_t`. That would be more consistent with the constructor parameter types, and accessor method types.

I was just thinking the same thing.